app/javascript/src/Navigation/components/ContextSelector.jsx

```diff
@@ -68,16 +68,16 @@ class ContextSelector extends React.Component<Props, State> {
     return 'PopNavigation-link'
   }
 
-  getClassNamesForService (service: Service): string {
+  getClassNamesForService (api: Api): string {
     const { activeMenu, currentApi } = this.props
     let classNames = 'PopNavigation-link'
 
-    if (['serviceadmin', 'monitoring'].indexOf(activeMenu) !== -1 &&
-      service.id === currentApi.service.id) {
+    if (['serviceadmin', 'monitoring', 'backend_api'].indexOf(activeMenu) !== -1 &&
+      `${api.type}${api.id}` === `${currentApi.type}${currentApi.id}`) {
```

Comment:
Is it `name` unique among apis? If so, we could use that instead of creating a string. Otherwise we could add a `getUniqueID()` method and use it also for the `li`'s key.

Reply:
Not really unique, BackendApi and a Service could have a the same `system_name` and anyone can have the same `name`. We could create a `getUniqueID` tho, but since we are not applying any other actions on these, I reckon is not needed right now. Not a strong opinion tho

Reply:
Is not very important, but I think reusing a `getUniqueID()` method will be nicer